cpp/pagerank_module/algorithm/pagerank.cpp

```diff
@@ -0,0 +1,142 @@
+#include <algorithm>
+#include <numeric>
+#include <queue>
+
+#include "pagerank.hpp"
+
+namespace pagerank_alg {
+
+PageRankGraph::PageRankGraph(const std::uint64_t number_of_nodes, const std::uint64_t number_of_edges,
+                             const std::vector<std::pair<std::uint64_t, std::uint64_t>> &edges)
```

Comment:
Since this is repeated throughout the code, and its pretty verbose, maybe it makes sense to create alias for it since its pretty verbose 
e.g.
`using edge_pairs = std::vector<std::pair<std::uint64_t, std::uint64_t>>;`
and use it like that.
```suggestion
PageRankGraph::PageRankGraph(const std::uint64_t number_of_nodes, const std::uint64_t number_of_edges, const edge_pairs &edges)
```

Reply:
I refactored this using this guide, therefore extracting the information I expected to be used by API users

[Google Alias guide](https://google.github.io/styleguide/cppguide.html#Aliases)

Reply:
Looks good, it makes more sense to just alias `EdgePairt` than the whole vector.